include/wlr/render/gles2.h

```diff
@@ -30,8 +30,19 @@ struct wlr_gles2_texture_attribs {
 	bool has_alpha;
 };
 
+struct wlr_buffer;
+struct wlr_gles2_buffer_attribs {
+	struct wlr_buffer *buffer;
+
+	GLuint rbo;
```

Comment:
I'm wondering whether the `rbo` is going to be useful for compositors.

Reply:
I am also not need the RBO, maybe remove it from wlr_gles2_buffer_attribs ?